### ISSUE TITLE
Support for Android embedding v2

### ISSUE THREAD

**Issue by Boehrsi** [rmtmckenzie/flutter_qr_mobile_vision#114]
Hi, as soon as an app uses the Flutter 1.12.x + Android v2 embedding the plugin stops functioning. The reason is the not available `registrar` if v2 embedding is used. The result is the following:

```
I/flutter ( 8156): MissingPluginException(No implementation found for method start on channel com.github.rmtmckenzie/qr_mobile_vision)
I/flutter ( 8156): MissingPluginException(No implementation found for method stop on channel com.github.rmtmckenzie/qr_mobile_vision)
```

With Android v2 embedding the registrar isn't used anymore and instead the [FlutterPlugin](https://github.com/flutter/engine/blob/master/shell/platform/android/io/flutter/embedding/engine/plugins/FlutterPlugin.java) + [ActivityPluginBinding interface](https://github.com/flutter/engine/blob/master/shell/platform/android/io/flutter/embedding/engine/plugins/activity/ActivityPluginBinding.java) should get used.

A complete guide can be found here: https://flutter.dev/docs/development/packages-and-plugins/plugin-api-migration.

It would be really nice to support the new embedding method and as this is pretty urgent for me I already prepared a branch. The corresponding PR will be opened in the next minutes.

Btw. thanks for the nice plugin :+1:.

**Comment by Boehrsi** [comment]
PR was added.

**Comment by rmtmckenzie** [comment]
This is fixed in 1.0.0. If you need to use 0.* because of its support for firebase, I recommend using the commit directly (or open a new bug and I'll try to put out a hotfix release for 0.*)
